scripts/get_s3_permissions.py

```diff
@@ -87,6 +109,8 @@ def download_files(s3_client, bucket_name, local_path, file_names, folders):
         file_path.parent.mkdir(parents=True, exist_ok=True)
         s3_client.download_file(bucket_name, file_name, str(file_path))
 
+    add_test_files("K6PerformanceTest", "Y05868.json", local_path)
```

Comment:
Is the path correct here? Shouldn't it be something like `layer/test_permissions/Y05868-TestApp-12345678/Y05868.json`?